app/Models/Patreon.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Models;
+
+use App\Services\Patreon\Resources\User;
+use Illuminate\Database\Eloquent\Model;
+use Spatie\MediaLibrary\HasMedia\HasMedia;
+use Spatie\MediaLibrary\HasMedia\HasMediaTrait;
+
+class Patreon extends Model implements HasMedia
```

Comment:
`Patreon` is named to generic as there are multiple concepts in `Patreon`. Let's rename this to `PatreonPleder`.